spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/config/KafkaBinderDefaultProperties.java

```diff
@@ -25,18 +25,12 @@
 @ConfigurationProperties(value = "spring.cloud.stream.binder.kafka.default")
 public class KafkaBinderDefaultProperties {
```

Comment:
If the reason is not to conflict with the `broker` specific properties such as `KafkaBinderConfigurationProperties`, may be we need to name the broker specific properties as `KafkaBrokerProperties` with the prefix `spring.cloud.stream.broker.kafka`? and this class can be `KafkaBinderProperties` with the prefix `spring.cloud.stream.binder.kafka`?